profile/README.md

```diff
@@ -1,6 +1,8 @@
 
 # Welcome to Confidential Computing API Group
 
+Collaboration at [slack](https://app.slack.com/huddle/T070GF9PKTQ/C0708HZ9087)
```

Comment:
I'm not sure if this is the correct link. It seems a cc-api huddle but not the channel.

Reply:
fixed at https://github.com/cc-api/.github/pull/1/commits/d09b6e071a9ef80883684d4f7a0b07492f1b14f5